src/sagemaker/lineage/query.py

```diff
@@ -207,29 +207,84 @@ class DashVisualizer(object):
     def __init__(self):
         """Init for DashVisualizer."""
         # import visualization packages
-        self.cyto, self.JupyterDash, self.html = self._import_visual_modules()
+        (
+            self.cyto,
+            self.JupyterDash,
+            self.html,
+            self.Input,
+            self.Output,
+        ) = self._import_visual_modules()
+
+        self.entity_color = {
```

Comment:
I don't think this needs to be done now, especially since we'll likely need to add more custom logic beyond this and we don't have an existing alternative use case, but just a thought. 

If we ever had a case in the future where we wanted to have similar node graphs but with different data we couldn't reuse this class as written because it has stuff specific to the lineage case. However if we 1) inject this data into the DashVisualizer constructor and 2) slightly change the format data to also include values for the startArn selector (maybe needing an extra parameter to differentiate between showing a color vs the shape), we would have a DashVisualizer class that could create a graph of connections/nodes with any keys we want

ex:
```
lineage_graph = {
           "TrialComponent":  { "style": {"background-color": "#f6cf61"}, "isShape: false"}
            "Context": { "style": {"background-color": "#ff9900"}},
            "Action": { "style": {"background-color": "#88c396"}},
            "Artifact": { "style": {"background-color": "#146eb4"}},
            "startarn": { "style": {"shape": "star"}, "shape": "star"}},
}

pipelines_graph = {
 "TrainingJob": { "style": {"shape": "circle"}, "shape": "circle"}},
 "ProcessingJob": { "style": {"shape": "square"}, "shape": "square"}},
}

new DashVisualizer(lineage_graph);

new DashVisualizer(pipelines_graph);

```

Reply:
injected `lineage_graph` data to `DashVisulaizer` class